charts/hivemq-platform/values.yaml

```diff
@@ -539,6 +548,9 @@ extensions:
 #    priority: 1000
 #    # Dynamically configures the extension start-priority value in the "hivemq-extension.xml" file.
 #    startPriority: 1000
+#    # Configures if certificate validation and hostname verification should be skipped for downloading the custom extension.
+#    skipHttpsCertificateValidation: false
+#    skipHttpsHostnameVerification: false
```

Comment:
Maybe I would include this commented out code in all of the extension defined here so they're all aligned, and as a different example, I would set them to `true` for the custom extension one.

Reply:
Discussed via Slack, we leave it as is.